src/rpc/blockchain.cpp

```diff
@@ -236,6 +236,7 @@ UniValue blockToJSON(const CBlock& block, const CBlockIndex* blockindex, bool tx
     result.pushKV("blockcommitments", blockindex->hashBlockCommitments.GetHex());
     result.pushKV("authdataroot", blockindex->hashAuthDataRoot.GetHex());
     result.pushKV("finalsaplingroot", blockindex->hashFinalSaplingRoot.GetHex());
+    result.pushKV("finalorchardroot", blockindex->hashFinalOrchardRoot.GetHex());
```

Comment:
Document this in the RPC help text for `getblock`.

Reply:
For pre-NU5 it will return the null hash (i.e. `[0; 32]` in hex).

Reply:
It can also be null if the block has never been connected to a main chain, so it's easier to just always show it and require the caller to filter out null hashes.

Reply:
Eh, actually we hold `cs_main` inside `blockToJSON`, so I can hide it from pre-NU5 blocks.